torch/csrc/autograd/FunctionsManual.cpp

```diff
@@ -3266,6 +3267,12 @@ Tensor constant_pad_nd_backward(const Tensor& grad, IntArrayRef pad) {
   return at::constant_pad_nd(grad, negated_pad, 0);
 }
 
+at::Tensor pad_backward(const at::Tensor& grad, const at::Tensor& self, const at::Tensor& pad_width, c10::string_view mode_str) {
+  Tensor pad_width_ = at::native::expand_pad_specifier_arg(pad_width, "pad_width", self.dim());
```

Comment:
You should not call into `at::native` from here.
This function is supposed to be differentiable if it is in this file.